index.d.ts

```diff
@@ -1,17 +1,20 @@
-import { Dispatch, Store, Unsubscribe } from 'redux';
+import { Dispatch, Store, Unsubscribe, AnyAction } from 'redux';
 
 export function shallowEquals(a: any, b: any): boolean;
 
-interface Options {
+type Options = {
   skipInitialCall?: boolean;
   equals?: typeof shallowEquals;
 }
 
-type Observer<S = any, MS = any> = unknown;
-type Mapper<S = any, MS = any> = (state: S) => MS;
-type Dispatcher<MS> = (dispatch: Dispatch<MS>, currentState: MS, previousState: MS) => unknown;
+type Mapper<S, MS> = (state: S) => MS;
```

Comment:
I'm not too sure this is correct. A `mapper` always takes a `store` state and can return any value. Is there a way to enforce `S` to be a POJO? Is it necessary?

I feel that `S = any, MS = any` is too loose and insufficiently descriptive. I may be wrong, though. (First time writing any TypeScript here.)

(By the way, `MS` stands for "MappedState")

Reply:
`state` is not necessarily an object if some one really wants that way, so we might leave it

Reply:
state is not necessarily an object if someone really wants, we might leave it to be loose